perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -33,12 +33,49 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+
+STANDARD = 'standard'
+GP2 = 'gp2'
+IO1 = 'io1'
+
 DISK_TYPE = {
-    disk.STANDARD: 'standard',
-    disk.REMOTE_SSD: 'gp2',
-    disk.PIOPS: 'io1'
+    # no need to translate 'standard' because the old and new names
```

Comment:
As discussed, let's retain the default behavior of provisioning a "standard" scratch disk. i.e., ./pkb.py --benchmarks=xyz --scratch_disk_size=100G will provision a 100GB data disk for the user with a default "standard" disk type without having to ask the user to figure out what exactly the "standard" disk type string is for the provider they want to run on.